components/FlexibleHeader/src/MDCFlexibleHeaderViewController.h

```diff
@@ -40,6 +40,14 @@
 /** The flexible header view instance that this controller manages. */
 @property(nonatomic, strong, nonnull, readonly) MDCFlexibleHeaderView *headerView;
 
+/**
+ A block that is invoked when the FlexibleHeaderViewController receives a call to @c
```

Comment:
Nit: Naming: ...when the MDCFlexibleHeaderViewController receives...

Reply:
Done